tests/e2e/specs/gscan.cy.js

```diff
@@ -192,6 +192,18 @@ describe('GScan component', () => {
     })
   })
 
+  describe('Task waiting badges', () => {
+    it('displays retry and held icons', () => {
+      cy.get('.c-gscan')
+        .find('[data-node-name="one"]').as('parent')
+        .find('.node:first .task-state-badge:first')
+        .should('have.class', 'held')
```

Comment:
Is this too trivial a check, or is this a case of "at some level you have to trust the language"?